snuba/request.py

```diff
@@ -64,7 +65,7 @@ def build_with_extensions(cls, extensions: Mapping[str, QueryExtension]) -> Requ
         }
         return cls(generic_schema, extensions_schemas)
 
-    def validate(self, value) -> Request:
+    def validate(self, value) -> Tuple[Request, Mapping[str, Mapping[str, Any]]]:
```

Comment:
It'd be helpful to have a docstring here that explains the return type at this point, since it's not particularly self-evident.